manifests/params.pp

```diff
@@ -67,7 +67,7 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
-  $lazy_enabled = false
+  $manage_squid = true
```

Comment:
If lazy is simply just "on" in Pulp now, and it requires a proxy like this squid setup to be configured then having this set to true by default is fine. I equate this to how it manages and configures mongo and apache by default.